libs/cypress-indexeddb/src/lib/object-store-CRUD.ts

```diff
@@ -35,6 +46,17 @@ export function readItem<T>(store: IDBObjectStore, key: string): Promise<T> {
     });
 }
 
+/**
+ * Delete item for the provided store key
+ *
+ * @remarks The `deleteItem` method deletes the value of the provided key, or undefined if it does not exist. You can chain assertions from this method. If you use TypeScript, you can set the type of the returned value
+ *
+ * @param store `IDBObjectStore` instance
+ * @param key
+ *
+ * @returns IDBObjectStore
+ * @throws {Error} If the connections fails to open or the read operation fails.
```

Comment:
technically it only throws an error if it is chained off from a method that does not return an object store.